src/screens/MinigameScreen.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { StyleSheet, ImageBackground, View } from 'react-native';
+import BestGameEver from '../components/BestGameEver.js';
+
+const styles = StyleSheet.create({
```

Comment:
can you put `styles` into its own `minigame.js` file (we can try keeping one per component/screen) 
this way we can keep the styles and reuse the style sheets we want.

so if its global styles (page container, etc) then put it in` src/styles/global.js`
if they are repeating variables (colors, etc) put into `src/styles/variables.js`